Added safety definition

zhongjun2 opened the pull request.

germonprez commented:
Hi @zhongjun2 

Thanks for the PR. A few comments: 

1) Could you add a DCO sign-off for the PR
2) Could you place the markdown file in a /definition folder 
3) I think you just include the metric and link to the metric in the Metrics in the Model section

zhongjun2 commented:
> Hi @zhongjun2
> 
> Thanks for the PR. A few comments:
> 
> 1. Could you add a DCO sign-off for the PR
> 2. Could you place the markdown file in a /definition folder
> 3. I think you just include the metric and link to the metric in the Metrics in the Model section

Done

> Hi @zhongjun2
> 
> Thanks for the PR. A few comments:
> 
> 1. Could you add a DCO sign-off for the PR
> 2. Could you place the markdown file in a /definition folder
> 3. I think you just include the metric and link to the metric in the Metrics in the Model section

Done

eyehwan commented:
@zhongjun2 Does this model include data insight?

zhongjun2 commented:
> @zhongjun2 Does this model include data insight?

The document does not yet present the data and associated analysis

eyehwan commented on metrics-model-libs/safety/definition/safety.md:
```diff
@@ -0,0 +1,202 @@
+# Data Insights 
+
+## Background of the Investigated Context
+We choose a Linux OS distribution community to show metric model. 
```
What does this mean?

zhongjun2 replied:
done

eyehwan commented on metrics-model-libs/safety/definition/safety.md:
```diff
@@ -0,0 +1,202 @@
+We choose a Linux OS distribution community to show metric model. 
+
+## Insights Drawn from the Metrics Model
+Provide text and visualizations that describe how the unified metrics model is used to draw insights from the evaluated community or communities. 
```
if it doesn't privde insight, I think we should move it.

eyehwan commented on metrics-model-libs/safety/definition/safety.md:
```diff
@@ -0,0 +1,202 @@
+
+
+# References
+https://docs.google.com/document/d/1DpHhVMAwucaKoXoOYMCnmxZuhWMqWupDsZvWpxe_A0M/edit
```
this is the draft for review, not references. Papers, books, articles are suitable here.

zhongjun2 replied:
done

germonprez commented:
Hi @zhongjun2 -- Thanks for continuing to move this forward. A few more suggestions: 

1) Could you simply provide a link to the metric used in the metric model? I don't think that we need the rich text below the link to the metric. 
2) You could likely follow the suggestions from @eyehwan to remove the headers that are not being used. We can return to include them when we have a sample community to talk about.

zhongjun2 commented:
> Hi @zhongjun2 -- Thanks for continuing to move this forward. A few more suggestions:
> 
> 1. Could you simply provide a link to the metric used in the metric model? I don't think that we need the rich text below the link to the metric.
> 2. You could likely follow the suggestions from @eyehwan to remove the headers that are not being used. We can return to include them when we have a sample community to talk about.

Added link to the metrics in the doc. Removed the headers that are not being used

sgoggins reviewed (approved):
LGTM

eyehwan commented:
Looks good to me!
